specs/pvs/corla.pvs

```diff
@@ -40,16 +85,14 @@
 %%% informal specifications which will be extracted using a shell script
 %%% into a well-typed FAFESSL informal domain model specification.
 %%%
-%%% As mentioned above, we write formal specifications using a
-%%% literate style, a la Knuth. Doing so permits us to document our
```

Comment:
"literate" and "literate programming" are technical terms, but don't really sound like it. Thus I think it makes sense to retain the reference to Knuth, or perhaps, better, something like https://en.wikipedia.org/wiki/Literate_programming, to help clarify for readers what is going on.